### PR TITLE
LPS-201506 & LPS-201822: Replace AUI() usages in commerce-subscription-web and commerce-theme-speedwell

### PR DESCRIPTION
**Pull request by DiegoHu97**
[Link to ticket](https://liferay.atlassian.net/browse/LPS-201506)

## Steps 

**edit_definition_subscription_info.jsp**

- Go to Control Panel > Sites > Create a Minium site
- Go to Control Menu > Commerce > Product Management > Products
- Edit a Product > go to Subscription tab

https://github.com/liferay-echo/liferay-portal/assets/91208451/4202254e-230b-467d-9801-dc343df4d7b6

**edit_instance_subscription_info.jsp**

- Go to Control Panel > Sites > Create a Minium site
- Go to Control Menu > Commerce > Product Management > Products
- Edit a Product > go to SKUs tab > Edit an item 
- Go to Subscriptions tab

https://github.com/liferay-echo/liferay-portal/assets/91208451/c60bd143-9b5d-41db-b67a-0fbbcb62ce19

**general.jsp**
I have not found this in the UI, but the use case is the same as the two before, so I think there is no need to find it

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by DiegoHu97** [comment]
ci:test:sf

**Comment by DiegoHu97** [comment]
ci:test:sf

**Comment by DiegoHu97** [comment]
ci:test:sf

**Comment by DiegoHu97** [comment]
ci:test:sf

File: modules/apps/commerce/commerce-subscription-web/src/main/resources/META-INF/resources/commerce_subscription_entry/general.jsp [hunk 1]
```diff
@@ -142,11 +142,11 @@ if (deliveryMaxSubscriptionCycles > 0) {
 					</aui:select>
 
 					<div class="never-ends-header">
-						<aui:input checked="<%= ending ? false : true %>" name="neverEnds" type="toggle-switch" />
+						<aui:input checked="<%= ending ? false : true %>" name="neverEnds" onClick='<%= liferayPortletResponse.getNamespace() + "neverEndsToggle();" %>' type="toggle-switch" />
 					</div>
 
-					<div class="never-ends-content">
-						<aui:input disabled="<%= ending ? false : true %>" helpMessage="max-subscription-cycles-help" label="end-after" name="maxSubscriptionCycles" suffix='<%= LanguageUtil.get(request, "cycles") %>' value="<%= String.valueOf(maxSubscriptionCycles) %>">
+					<div class="<%= ending ? "never-ends-content" : "never-ends-content hide" %>">
```
Comment (sandrodw3) [inline modules/apps/commerce/commerce-subscription-web/src/main/resources/META-INF/resources/commerce_subscription_entry/general.jsp:148]: Here you can put always `never-ends-content` and only conditionate `hide`: `class="never-ends-content <%= ending ? StringPool.BLANK : "hide" %>"`

File: modules/apps/commerce/commerce-subscription-web/src/main/resources/META-INF/resources/commerce_subscription_entry/general.jsp [hunk 2]
```diff
@@ -452,69 +448,39 @@ if (deliveryMaxSubscriptionCycles > 0) {
...
-				}
-				else {
-					A.one(
-						'#<portlet:namespace />deliveryMaxSubscriptionCycles'
-					).attr('disabled', true);
-				}
-			},
-		},
-	});
+<aui:script>
```
Comment (sandrodw3) [inline modules/apps/commerce/commerce-subscription-web/src/main/resources/META-INF/resources/commerce_subscription_entry/general.jsp:474]: Could we change this to a simple `<script>`? And the same with others that doesn't use aui

File: modules/apps/commerce/commerce-subscription-web/src/main/resources/META-INF/resources/definition/edit_definition_subscription_info.jsp [hunk 3]
```diff
@@ -242,11 +242,11 @@ if (deliveryMaxSubscriptionCycles > 0) {
 
 		<div id="<portlet:namespace />deliveryNeverEndsContainer">
 			<div class="never-ends-header">
-				<aui:input checked="<%= deliveryEnding ? false : true %>" label="never-ends" name="deliveryNeverEnds" type="toggle-switch" />
+				<aui:input checked="<%= deliveryEnding ? false : true %>" label="never-ends" name="deliveryNeverEnds" onClick='<%= liferayPortletResponse.getNamespace() + "deliveryNeverEndsToggle();" %>' type="toggle-switch" />
 			</div>
 
-			<div class="never-ends-content">
-				<aui:input disabled="<%= deliveryEnding ? false : true %>" helpMessage="max-subscription-cycles-help" label="end-after" name="deliveryMaxSubscriptionCycles" suffix='<%= LanguageUtil.get(request, "cycles") %>' value="<%= String.valueOf(deliveryMaxSubscriptionCycles) %>">
+			<div class="<%= ending ? "never-ends-content" : "never-ends-content hide" %>">
```
Comment (sandrodw3) [inline modules/apps/commerce/commerce-subscription-web/src/main/resources/META-INF/resources/definition/edit_definition_subscription_info.jsp:248]: Same here with the condition

File: modules/apps/commerce/commerce-subscription-web/src/main/resources/META-INF/resources/definition/edit_definition_subscription_info.jsp [hunk 4]
```diff
@@ -321,107 +318,76 @@ if (deliveryMaxSubscriptionCycles > 0) {
...
 </aui:script>
 
-<aui:script use="liferay-form">
-	A.one('#<portlet:namespace />neverEnds').on('change', (event) => {
-		var formValidator = Liferay.Form.get('<portlet:namespace />fm')
-			.formValidator;
-
-		formValidator.validateField('<portlet:namespace />maxSubscriptionCycles');
-	});
+<aui:script>
```
Comment (sandrodw3) [inline modules/apps/commerce/commerce-subscription-web/src/main/resources/META-INF/resources/definition/edit_definition_subscription_info.jsp:353]: Same with the `<script>`

File: modules/apps/commerce/commerce-subscription-web/src/main/resources/META-INF/resources/definition/edit_instance_subscription_info.jsp [hunk 5]
```diff
@@ -162,11 +162,11 @@ if (deliveryMaxSubscriptionCycles > 0) {
 
 				<div id="<portlet:namespace />neverEndsContainer">
 					<div class="never-ends-header">
-						<aui:input checked="<%= ending ? false : true %>" name="neverEnds" type="toggle-switch" />
+						<aui:input checked="<%= ending ? false : true %>" name="neverEnds" onClick='<%= liferayPortletResponse.getNamespace() + "neverEndsToggle();" %>' type="toggle-switch" />
 					</div>
 
-					<div class="never-ends-content">
-						<aui:input disabled="<%= ending ? false : true %>" helpMessage="max-subscription-cycles-help" label="end-after" name="maxSubscriptionCycles" suffix='<%= LanguageUtil.get(request, "cycles") %>' value="<%= String.valueOf(maxSubscriptionCycles) %>">
+					<div class="<%= ending ? "never-ends-content" : "never-ends-content hide" %>">
```
Comment (sandrodw3) [inline modules/apps/commerce/commerce-subscription-web/src/main/resources/META-INF/resources/definition/edit_instance_subscription_info.jsp:168]: Same 😛

**Review by sandrodw3 — COMMENTED** [review]
@DiegoHu97 some small improvements 🥳

**Comment by DiegoHu97** [comment]
@sandrodw3 Updated! Thanks for the review 😄

**Comment by sandrodw3** [comment]
ci:forward

**Comment by sandrodw3** [comment]
ci:stop

**Comment by DiegoHu97** [comment]
ci:test:commerce-functional

**Comment by sandrodw3** [comment]
ci:test:relevant

**Comment by sandrodw3** [comment]
ci:test:relevant

**Comment by DiegoHu97** [comment]
ci:test:relevant

**Comment by ealonso** [comment]
ci:test:relevant

**Comment by ealonso** [comment]
ci:forward:force

**Pull request closed without merging**